src/Std/Data/OrderAxioms/TransOrd.lean

```diff
@@ -0,0 +1,166 @@
+/-
+Copyright (c) 2024 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Markus Himmel
+-/
+prelude
+import Init.Data.Ord
+
+set_option autoImplicit false
+set_option linter.missingDocs true
+
+universe u
+
+namespace Std
+
+/--
+Class for functions `α → α → Ordering` which are oriented: flipping the arguments amounts
+to applying `Ordering.swap` to the return value.
+ -/
+class OrientedCmp {α : Type u} (cmp : α → α → Ordering) : Prop where
```

Comment:
Any particular reason why `ReflOrd` gets its own file but `OrientedOrd` does not?

Reply:
I decided to put all of the Cmp/Ord classes into one file called `Std/Classes/Ord.lean`. Perhaps we want to add some more classes in the future, hence the new directory.